simulator/src/canvas2svg.js

```diff
@@ -18,20 +18,20 @@
 
```

Comment:
This is a thirdparty lib, better not to change this file

Reply:
Okay which files should I not modify under simulator ?

Reply:
Everything except thirdparty scripts. there are only few such files in simulator folder, rest are moved to npm

Reply:
Ok then I am closing this PR.